raster/r.what/main.c

```diff
@@ -255,8 +273,13 @@ int main(int argc, char *argv[])
     Cats = Vect_new_cats_struct();
     G_get_window(&window);
 
+    if (strcmp(opt.format->answer, "json") == 0) {
+        root_value = json_value_init_array();
```

Comment:
You could check here that the root_value is not returned as Null.

```c
root_value = json_value_init_array();
if (root_value == NULL) {
    G_fatal_error(_("Failed to initialize JSON array. Out of memory?"));
}
```

Reply:
Will fix.

Reply:
Done.